src/main/java/mat/client/HarpSupportWidget.ui.xml

```diff
@@ -0,0 +1,24 @@
+<ui:UiBinder xmlns:ui='urn:ui:com.google.gwt.uibinder'
+             xmlns:g='urn:import:com.google.gwt.user.client.ui'>
+    <g:HTMLPanel>
+        <div id="harp-support-header-section" style="text-align: center; margin: 1em;">
+            <p>
+                <i class="fa fa-frown-o fa-3x" aria-hidden="true"></i>
+                <h2>We are sorry...</h2>
+            </p>
+        </div>
+        <div id="harp-support-links-section" style="text-align: left; margin: 2em;">
+            <h4>We were unable to access the Measure Authoring Tool (MAT). The following resources may be helpful:</h4>
+            <ul>
+                <li>View the <a href="https://harp.qualitynet.org/">HARP self-service</a> page</li>
+                <li>Submit a ticket via <a href="https://oncprojectracking.healthit.gov/support/login.jsp">the Office of the National Coordinator (ONC) Jira System</a> and follow the directions on the page
```

Comment:
This line is still ambiguous to me: `follow the directions on the page`. There are no direction to follow related to MAT on ONC jira. something is missing here. I don't think we need this `and follow the directions on the page`

Reply:
We will be updating this text, People are not going to submit any ONC jira ticket, if they are New users. A new user registration form is in development (Drupal)

Reply:
I see, thanks for clarification.

Reply:
That's fair. We'll have to work with Kathy, and a change will be needed in both develop and master. Going to close this PR as is so we can start testing.